docs/user-guide/data-reduction.md

```diff
@@ -0,0 +1,169 @@
+# Focused reflectometry data reduction
+
+The goal of the reflectometry data reduction is to compute the reflectivity $R(Q)$ of the sample as a function of the momentum transfer $Q$.
+
+Based on [J. Stahn, A. Glavic, Focusing neutron reflectometry: Implementation and experience on the TOF-reflectometer Amor](#reference).
```

Comment:
We use bibtex in the docs of essdiffraction and scippneutron. Can you use that here as well?
See 
- https://github.com/scipp/essdiffraction/blob/c9c29dd6155cdb0f9fd201e6c9d6923a754b1e37/docs/conf.py#L36
- https://github.com/scipp/essdiffraction/blob/c9c29dd6155cdb0f9fd201e6c9d6923a754b1e37/docs/conf.py#L293-L295
- https://github.com/scipp/essdiffraction/tree/main/docs/about